src/Authoring/cswinmd/build/cswinmd.targets

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="ProduceWinMDFromCSharpComponent" BeforeTargets="InsertComponentWinMD">
```

Comment:
Consider prefixing the targets with like CsWinRT or CsWinMD to avoid conflicts with other potential targets.  This is pretty specific not to be an issue, but InsertComponentWinMD can end up being used elsewhere too.